packages/frame-sdk-backend/scripts/build.js

```diff
@@ -0,0 +1,12 @@
+const esbuild = require('esbuild')
+
+esbuild.buildSync({
```

Comment:
we don't actually use or need esbuild

you can copy the same build setup from `frame-core`:
- https://github.com/farcasterxyz/frames/blob/main/packages/frame-core/package.json#L9-L11
- https://github.com/farcasterxyz/frames/blob/main/packages/frame-core/tsconfig.node.json

this will output node friendly builds